extra/errors.vim

```diff
@@ -1,11 +1,29 @@
 " Inspect code
+let g:WhichKeyDesc_Errors_InspectCode = "<leader>eL inspect-code"
 nnoremap <leader>eL    :action InspectCode<CR>
 vnoremap <leader>eL    :action InspectCode<CR>
 
+" List errors
+let g:WhichKeyDesc_Errors_ListErrors = "<leader>el list-errors"
+nnoremap <leader>el    :action CodeInspection.OnEditor<CR>
+vnoremap <leader>el    :action CodeInspection.OnEditor<CR>
+
+" Run inspection by name
+let g:WhichKeyDesc_Errors_RunInspectionByName = "<leader>eR run-inspection-by-name"
```

Comment:
Moved the added shortcuts from spacemacs/errors.vim to here.

This particular shortcut was previously assigned to `<leader>es` which conflicted with "Edit Inspection Settings" (line 22), so I opted to change it to `<leader>eR` instead.